ProteoformSuiteInternal/ProteoformRelation.cs

```diff
@@ -51,22 +51,83 @@ public List<ProteoformRelation> mass_difference_group
                 this.group_count = value.Count;
                 this.group_adjusted_deltaM = value.Select(r => r.delta_mass).Average();
                 _mass_difference_group = value;
+                this.accepted = set_accepted(); //dependent on peak count
             }
         }
-        public int unadjusted_group_count { get; set; }
+
+        public int unadjusted_group_count { get; set; } //this is never set to anything? 
+        public string accession { get; set; } //theoretical for ET, null for EE
+        public string name { get; set; } //theoretical for ET, null for EE
+        public string fragment { get; set; }
+        public string ptm_list { get; set; }
+        public double proteoform_mass_1 { get; set; } //experiment for ET, experiment_1 for EE
+        public double proteoform_mass_2 { get; set; } //theoretical for EE, experiment_2 for EE
+        public double agg_intensity_1 { get; set; } //experiment for ET, experiment_1 for EE
+        public double agg_intensity_2 { get; set; } //0 for ET, experiment_2 for EE
+        public double agg_RT_1 { get; set; } //experiment for ET, experiment_1 for EE
+        public double agg_RT_2 { get; set; } //0 for ET, experiment_2 for EE
+        public int num_observations_1 { get; set; }
+        public int num_observations_2 { get; set; } //0 for ET
+        public int lysine_count { get; set; }
         public double group_adjusted_deltaM { get; set; }
+        public int running_sum { get; set; }
         public int group_count { get; set; }
+        public bool outside_no_mans_land { get; set; }
 
-        public ProteoformRelation(Proteoform pf1, Proteoform pf2, ProteoformComparison relation_type, double delta_mass) : base(pf1, pf2, relation_type, delta_mass)
+        //et
+        public ProteoformRelation(ExperimentalProteoform pf1, TheoreticalProteoform pf2, ProteoformComparison relation_type, double delta_mass) : base(pf1, pf2, relation_type, delta_mass)
         {
             this.group_adjusted_deltaM = delta_mass;
+            this.proteoform_mass_1 = pf1.agg_mass;
+            this.proteoform_mass_2 = pf2.modified_mass;
+            this.agg_intensity_1 = pf1.agg_intensity;
+            this.agg_RT_1 = pf1.agg_rt;
+            this.lysine_count = pf1.lysine_count;
+            this.num_observations_1 = pf1.observation_count;
+            this.accession = pf2.accession;
+            this.fragment = pf2.fragment;
+            this.ptm_list = pf2.ptm_descriptions;
+            this.name = pf2.name;
+            this.outside_no_mans_land = set_outside_no_mans_land(); //only relations outside no-mans-land should be formed into peaks.
         }
+
+        //ee
+        public ProteoformRelation(ExperimentalProteoform pf1, ExperimentalProteoform pf2, ProteoformComparison relation_type, double delta_mass) : base(pf1, pf2, relation_type, delta_mass)
```

Comment:
Redundancy here could be reduced by type-checking for ET or EE. You could even just check which ProteoformComparison enum value is passed to make that easier.